Query/objects/ip_intelligence.json

```diff
@@ -0,0 +1,47 @@
+{
+	"description": "Insights from threat intelligence platforms about IP Addresses",
+	"caption": "IP Threat Intelligence",
+	"name": "ip_intelligence",
+	"attributes": {
+
+		"labels": {
+			"description": "The labels or tags in the intelligence."
+			"requirement": "optional",
+		},
+		"reputations": {
+			"description": "Reputation score as reported by provider",
+			"requirement": "optional",
+		},
+		"location": {
+			"requirement": "optional",
+		},
+		"ip": {
+			"requirement": "optional",
+		},
+		"subnet": {
+			"requirement": "optional",
+		},
+		"asn": {
+			"requirement": "optional",
+		},
+		"as_owner": {
+			"requirement": "optional",
+		},
```

Comment:
They are defined in `query/dictionary.json`.

I assume AS Owner is the registrant of the ASN. But it may make better sense to call it ASN Owner...

Example:
`"as_owner": "ATT-INTERNET4",`